common-config/src/main/kotlin/org/jeongmo/migration/common/auth/utils/WebClientUserInfoPropagationFilterFunction.kt

```
@@ -0,0 +1,31 @@
+import org.springframework.web.reactive.function.client.ClientRequest
+import org.springframework.web.reactive.function.client.ClientResponse
+import org.springframework.web.reactive.function.client.ExchangeFilterFunction
+import org.springframework.web.reactive.function.client.ExchangeFunction
+import reactor.core.publisher.Mono
+
+class WebClientUserInfoPropagationFilterFunction: ExchangeFilterFunction {
+
+    override fun filter(request: ClientRequest, next: ExchangeFunction): Mono<ClientResponse> {
+        val currentRequest = try {
+            (RequestContextHolder.getRequestAttributes() as? ServletRequestAttributes)?.request
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)